src/Controller/UserProfileController.php

```diff
@@ -696,9 +696,10 @@ public function updateInfo(Request $request, Response $response, array $args)
         $currentUser = $this->ci->currentUser;
 
         // Access-controlled resource - check that currentUser has permission to edit submitted fields for this user
-        if (!$authorizer->checkAccess($currentUser, 'update_user_field', [
+        if (!$authorizer->checkAccess($currentUser, 'update_user_field_less_restrictive', [
             'user' => $user,
-            'fields' => array_values(array_unique($fieldNames)),
+            //JMhack dynamicky pridavana pole
```

Comment:
I think some of your site changes made it in this PR ?

Reply:
Hi, Luis,

bug: 
$user->$name = $value;
this with Userfrosting 4.4 don't use slug table,

functional chage:
$user->fill($filled_data);

array $filled_data is in the middle

---
sorry for comment, it is internal.

Thanks for Userfrosting modules.
Josef